.github/workflows/ci.yml

```diff
@@ -118,7 +118,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         repository: maputnik/desktop
-        ref: master
+        ref: main
```

Comment:
```suggestion
        ref: master
```

Reply:
This is using the desktop repo (god knows why...) and the branch name there is still master